CHANGELOG.md

```diff
@@ -57,15 +57,18 @@ Converting from v3.x to v4.0
   compatibility, a joint's parent and child PhysicalFrames are swapped when
   opening a Model if the `reverse` element is set to `true`.
 - The `Manager::integrate(SimTK::State&)` call is deprecated and replaced by
-  `Manager::integrate(SimTK::State&, double)`. Here is a before-after example
-  (see the documentation in the `Manager` class for more details):
+  `Manager::integrate(double)`. You must also now call 
+  `Manager::initialize(SimTK::State&)` before integrating. Here is a 
+   before-after example (see the documentation in the `Manager` class 
+   for more details):
   - Before:
 	- manager.setInitialTime(0.0);
 	- manager.setFinalTime(1.0);
 	- manager.integrate(state);
   - After:
     - state.setTime(0.0);
-	- manager.integrate(state, 1.0);
+	- manager.initialize(state);
+	- manager.integrate(1.0);
```

Comment:
It is not clear now that the *state* passed into `Manager::initialize(state)` is operated upon by `Manager::integrate(finalT)`.

Also, is one now able to update the *state* following integration and prior to subsequent integration (as in #1662) by inserting `Manager::initialize(state)` prior to `integrate`? Might indicate that issue and your fix here or in bug fixes below.

Reply:
Don't think that should be possible, so maybe more documentation about that to be clearer?